src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,180 @@
+        arrayObj = stage2(arrayObj);
+        return arrayObj.clone();
+    }
+
+    /**
+     * The second part.
+     *
+     * @param rango int[].
+     * @return arrayObj2 Object[].
+     */
+    public Object[] fizzBuzz2(final int[] rango) {
+        Object[] fizz2 = new Object[]{};
+        fizz2 = fizzBuzz(rango);
+        fizz2 = stage3(fizz2);
+        fizz2 = stage4(fizz2);
+        fizz2 = stage5(fizz2);
+        fizz2 = stage6(fizz2);
+        return fizz2;
+    }
+
+    /**
+     * Fill the arrangement with numbers.
+     *
+     * @param ini      int
+     * @param arrayObj Object[].
+     * @return Object[].
+     */
+    private Object[] fill(final int ini, final Object[] arrayObj) {
+        int c = ini;
+        for (int i = 0; i < arrayObj.length; i++) {
+            arrayObj[i] = c;
+            c++;
+        }
+        return arrayObj;
+    }
+
+    /**
+     * Query.
+     *
+     * @param value int[].
+     * @return boolean.
+     */
+    private boolean query(final Object value) {
+        return value.equals("FizzBuzz") || value.equals("Fizz") || value.equals("Buzz");
+    }
+
+    /**
+     * A number is fizz if it is divisible by 3 and 5.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage0(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC = array;
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_5 == 0 && (Integer) arrayC[i] % NUMBER_3 == 0 ? "FizzBuzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is fizz if it is divisible by 5.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage1(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_5 == 0 ? "Buzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+
+    /**
+     * A number is fizz if it is divisible by 3.
+     *
+     * @param array int[].
+     * @return Object[]
+     */
+    private Object[] stage2(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_3 == 0 ? "Fizz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is buzz if it has a 5 in it.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage3(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_10 == NUMBER_5 ? "Buzz"
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)